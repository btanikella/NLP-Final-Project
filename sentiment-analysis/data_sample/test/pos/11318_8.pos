DT NN VBD RB JJ : PRP RB VBZ RB JJ IN DT JJ NNS IN DT NN VBD RB CD NNS IN DT NN NN VBD JJ VBG DT NNS WDT MD VB VBN CC JJ JJ TO VB IN DT NN WRB JJ PRP VBP PRP VBP : PRP VBP RB
IN DT VBD VBN IN DT JJ NN : PRP MD VB JJ IN PRP MD VB VBN DT IN DT JJ NN NN CC RB IN DT NN NN WDT DT NNP VBD
PRP RB VBZ NN  IN DT VBP : DT NN NN IN DT NNS MD RB VB : CC PRP VBD IN DT NN NN NN WDT MD VB VBN IN DT JJ NNS CC MD RB IN PRP VBZ NNS WDT VBZ RB VBN TO VB VBN
DT JJ NN IN WDT DT NNP VBD DT JJ NNS CC RB  NNP VBZ RB VB DT JJ NN DT RBR  JJ TO DT NN IN NNP CD CD
