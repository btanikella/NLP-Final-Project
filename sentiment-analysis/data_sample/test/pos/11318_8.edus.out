Sentence #1 (54 tokens):
This movie was really interesting... it also is quite shocking as the similar events of the movie occurred only 10 months after the movie premiered.it was interesting seeing the problems that could be encountered and realistic enough to show that no matter how prepared you think you are - you aren't.
[Text=This CharacterOffsetBegin=4 CharacterOffsetEnd=8 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=9 CharacterOffsetEnd=14 PartOfSpeech=NN] [Text=was CharacterOffsetBegin=15 CharacterOffsetEnd=18 PartOfSpeech=VBD] [Text=really CharacterOffsetBegin=19 CharacterOffsetEnd=25 PartOfSpeech=RB] [Text=interesting CharacterOffsetBegin=26 CharacterOffsetEnd=37 PartOfSpeech=JJ] [Text=... CharacterOffsetBegin=37 CharacterOffsetEnd=40 PartOfSpeech=:] [Text=it CharacterOffsetBegin=41 CharacterOffsetEnd=43 PartOfSpeech=PRP] [Text=also CharacterOffsetBegin=44 CharacterOffsetEnd=48 PartOfSpeech=RB] [Text=is CharacterOffsetBegin=49 CharacterOffsetEnd=51 PartOfSpeech=VBZ] [Text=quite CharacterOffsetBegin=52 CharacterOffsetEnd=57 PartOfSpeech=RB] [Text=shocking CharacterOffsetBegin=58 CharacterOffsetEnd=66 PartOfSpeech=JJ] [Text=as CharacterOffsetBegin=67 CharacterOffsetEnd=69 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=70 CharacterOffsetEnd=73 PartOfSpeech=DT] [Text=similar CharacterOffsetBegin=74 CharacterOffsetEnd=81 PartOfSpeech=JJ] [Text=events CharacterOffsetBegin=82 CharacterOffsetEnd=88 PartOfSpeech=NNS] [Text=of CharacterOffsetBegin=89 CharacterOffsetEnd=91 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=92 CharacterOffsetEnd=95 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=96 CharacterOffsetEnd=101 PartOfSpeech=NN] [Text=occurred CharacterOffsetBegin=102 CharacterOffsetEnd=110 PartOfSpeech=VBD] [Text=only CharacterOffsetBegin=111 CharacterOffsetEnd=115 PartOfSpeech=RB] [Text=10 CharacterOffsetBegin=116 CharacterOffsetEnd=118 PartOfSpeech=CD] [Text=months CharacterOffsetBegin=119 CharacterOffsetEnd=125 PartOfSpeech=NNS] [Text=after CharacterOffsetBegin=126 CharacterOffsetEnd=131 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=132 CharacterOffsetEnd=135 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=136 CharacterOffsetEnd=141 PartOfSpeech=NN] [Text=premiered.it CharacterOffsetBegin=142 CharacterOffsetEnd=154 PartOfSpeech=NN] [Text=was CharacterOffsetBegin=155 CharacterOffsetEnd=158 PartOfSpeech=VBD] [Text=interesting CharacterOffsetBegin=159 CharacterOffsetEnd=170 PartOfSpeech=JJ] [Text=seeing CharacterOffsetBegin=171 CharacterOffsetEnd=177 PartOfSpeech=VBG] [Text=the CharacterOffsetBegin=178 CharacterOffsetEnd=181 PartOfSpeech=DT] [Text=problems CharacterOffsetBegin=182 CharacterOffsetEnd=190 PartOfSpeech=NNS] [Text=that CharacterOffsetBegin=191 CharacterOffsetEnd=195 PartOfSpeech=WDT] [Text=could CharacterOffsetBegin=196 CharacterOffsetEnd=201 PartOfSpeech=MD] [Text=be CharacterOffsetBegin=202 CharacterOffsetEnd=204 PartOfSpeech=VB] [Text=encountered CharacterOffsetBegin=205 CharacterOffsetEnd=216 PartOfSpeech=VBN] [Text=and CharacterOffsetBegin=217 CharacterOffsetEnd=220 PartOfSpeech=CC] [Text=realistic CharacterOffsetBegin=221 CharacterOffsetEnd=230 PartOfSpeech=JJ] [Text=enough CharacterOffsetBegin=231 CharacterOffsetEnd=237 PartOfSpeech=JJ] [Text=to CharacterOffsetBegin=238 CharacterOffsetEnd=240 PartOfSpeech=TO] [Text=show CharacterOffsetBegin=241 CharacterOffsetEnd=245 PartOfSpeech=VB] [Text=that CharacterOffsetBegin=246 CharacterOffsetEnd=250 PartOfSpeech=IN] [Text=no CharacterOffsetBegin=251 CharacterOffsetEnd=253 PartOfSpeech=DT] [Text=matter CharacterOffsetBegin=254 CharacterOffsetEnd=260 PartOfSpeech=NN] [Text=how CharacterOffsetBegin=261 CharacterOffsetEnd=264 PartOfSpeech=WRB] [Text=prepared CharacterOffsetBegin=265 CharacterOffsetEnd=273 PartOfSpeech=JJ] [Text=you CharacterOffsetBegin=274 CharacterOffsetEnd=277 PartOfSpeech=PRP] [Text=think CharacterOffsetBegin=278 CharacterOffsetEnd=283 PartOfSpeech=VBP] [Text=you CharacterOffsetBegin=284 CharacterOffsetEnd=287 PartOfSpeech=PRP] [Text=are CharacterOffsetBegin=288 CharacterOffsetEnd=291 PartOfSpeech=VBP] [Text=- CharacterOffsetBegin=292 CharacterOffsetEnd=293 PartOfSpeech=:] [Text=you CharacterOffsetBegin=294 CharacterOffsetEnd=297 PartOfSpeech=PRP] [Text=are CharacterOffsetBegin=298 CharacterOffsetEnd=301 PartOfSpeech=VBP] [Text=n't CharacterOffsetBegin=301 CharacterOffsetEnd=304 PartOfSpeech=RB] [Text=. CharacterOffsetBegin=304 CharacterOffsetEnd=305 PartOfSpeech=.] 
root(ROOT-0, interesting-5)
det(movie-2, This-1)
nsubj(interesting-5, movie-2)
cop(interesting-5, was-3)
advmod(interesting-5, really-4)
punct(interesting-5, ...-6)
nsubj(shocking-11, it-7)
advmod(shocking-11, also-8)
cop(shocking-11, is-9)
advmod(shocking-11, quite-10)
parataxis(interesting-5, shocking-11)
prep(shocking-11, as-12)
det(events-15, the-13)
amod(events-15, similar-14)
pobj(as-12, events-15)
prep(events-15, of-16)
det(movie-18, the-17)
pobj(of-16, movie-18)
dep(shocking-11, occurred-19)
quantmod(10-21, only-20)
num(months-22, 10-21)
npadvmod(after-23, months-22)
prep(occurred-19, after-23)
det(premiered.it-26, the-24)
nn(premiered.it-26, movie-25)
nsubj(interesting-28, premiered.it-26)
cop(interesting-28, was-27)
pcomp(after-23, interesting-28)
xcomp(interesting-28, seeing-29)
det(problems-31, the-30)
dobj(seeing-29, problems-31)
nsubjpass(encountered-35, that-32)
aux(encountered-35, could-33)
auxpass(encountered-35, be-34)
rcmod(problems-31, encountered-35)
cc(encountered-35, and-36)
amod(enough-38, realistic-37)
conj(encountered-35, enough-38)
aux(show-40, to-39)
infmod(enough-38, show-40)
dep(show-40, that-41)
det(matter-43, no-42)
dep(that-41, matter-43)
advmod(prepared-45, how-44)
dep(think-47, prepared-45)
nsubj(think-47, you-46)
rcmod(matter-43, think-47)
nsubj(are-52, you-48)
dep(are-52, are-49)
punct(are-52, --50)
nsubj(are-52, you-51)
ccomp(think-47, are-52)
neg(are-52, n't-53)
punct(interesting-5, .-54)

Sentence #2 (35 tokens):
if this was made for an American audience - it would be different because they would have used this as a full propaganda film and not as a wake call which the BBC did!
[Text=if CharacterOffsetBegin=310 CharacterOffsetEnd=312 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=313 CharacterOffsetEnd=317 PartOfSpeech=DT] [Text=was CharacterOffsetBegin=318 CharacterOffsetEnd=321 PartOfSpeech=VBD] [Text=made CharacterOffsetBegin=322 CharacterOffsetEnd=326 PartOfSpeech=VBN] [Text=for CharacterOffsetBegin=327 CharacterOffsetEnd=330 PartOfSpeech=IN] [Text=an CharacterOffsetBegin=331 CharacterOffsetEnd=333 PartOfSpeech=DT] [Text=American CharacterOffsetBegin=334 CharacterOffsetEnd=342 PartOfSpeech=JJ] [Text=audience CharacterOffsetBegin=343 CharacterOffsetEnd=351 PartOfSpeech=NN] [Text=- CharacterOffsetBegin=352 CharacterOffsetEnd=353 PartOfSpeech=:] [Text=it CharacterOffsetBegin=354 CharacterOffsetEnd=356 PartOfSpeech=PRP] [Text=would CharacterOffsetBegin=357 CharacterOffsetEnd=362 PartOfSpeech=MD] [Text=be CharacterOffsetBegin=363 CharacterOffsetEnd=365 PartOfSpeech=VB] [Text=different CharacterOffsetBegin=366 CharacterOffsetEnd=375 PartOfSpeech=JJ] [Text=because CharacterOffsetBegin=376 CharacterOffsetEnd=383 PartOfSpeech=IN] [Text=they CharacterOffsetBegin=384 CharacterOffsetEnd=388 PartOfSpeech=PRP] [Text=would CharacterOffsetBegin=389 CharacterOffsetEnd=394 PartOfSpeech=MD] [Text=have CharacterOffsetBegin=395 CharacterOffsetEnd=399 PartOfSpeech=VB] [Text=used CharacterOffsetBegin=400 CharacterOffsetEnd=404 PartOfSpeech=VBN] [Text=this CharacterOffsetBegin=405 CharacterOffsetEnd=409 PartOfSpeech=DT] [Text=as CharacterOffsetBegin=410 CharacterOffsetEnd=412 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=413 CharacterOffsetEnd=414 PartOfSpeech=DT] [Text=full CharacterOffsetBegin=415 CharacterOffsetEnd=419 PartOfSpeech=JJ] [Text=propaganda CharacterOffsetBegin=420 CharacterOffsetEnd=430 PartOfSpeech=NN] [Text=film CharacterOffsetBegin=431 CharacterOffsetEnd=435 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=436 CharacterOffsetEnd=439 PartOfSpeech=CC] [Text=not CharacterOffsetBegin=440 CharacterOffsetEnd=443 PartOfSpeech=RB] [Text=as CharacterOffsetBegin=444 CharacterOffsetEnd=446 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=447 CharacterOffsetEnd=448 PartOfSpeech=DT] [Text=wake CharacterOffsetBegin=449 CharacterOffsetEnd=453 PartOfSpeech=NN] [Text=call CharacterOffsetBegin=454 CharacterOffsetEnd=458 PartOfSpeech=NN] [Text=which CharacterOffsetBegin=459 CharacterOffsetEnd=464 PartOfSpeech=WDT] [Text=the CharacterOffsetBegin=465 CharacterOffsetEnd=468 PartOfSpeech=DT] [Text=BBC CharacterOffsetBegin=469 CharacterOffsetEnd=472 PartOfSpeech=NNP] [Text=did CharacterOffsetBegin=473 CharacterOffsetEnd=476 PartOfSpeech=VBD] [Text=! CharacterOffsetBegin=476 CharacterOffsetEnd=477 PartOfSpeech=.] 
root(ROOT-0, different-13)
mark(made-4, if-1)
nsubjpass(made-4, this-2)
auxpass(made-4, was-3)
advcl(different-13, made-4)
prep(made-4, for-5)
det(audience-8, an-6)
amod(audience-8, American-7)
pobj(for-5, audience-8)
punct(different-13, --9)
nsubj(different-13, it-10)
aux(different-13, would-11)
cop(different-13, be-12)
mark(used-18, because-14)
nsubj(used-18, they-15)
aux(used-18, would-16)
aux(used-18, have-17)
advcl(different-13, used-18)
dobj(used-18, this-19)
prep(used-18, as-20)
det(film-24, a-21)
amod(film-24, full-22)
nn(film-24, propaganda-23)
pobj(as-20, film-24)
cc(used-18, and-25)
neg(as-27, not-26)
conj(used-18, as-27)
det(call-30, a-28)
nn(call-30, wake-29)
pobj(as-27, call-30)
dobj(did-34, which-31)
det(BBC-33, the-32)
nsubj(did-34, BBC-33)
rcmod(call-30, did-34)
punct(different-13, !-35)

Sentence #3 (50 tokens):
it still is propaganda, in some extend - no film today with these themes can not be - but it dealt with the issue successfully.a film that should be shown in all terrorism/counter-terrorism courses but will not because it shows faults which is not allowed to be acknowledged!
[Text=it CharacterOffsetBegin=482 CharacterOffsetEnd=484 PartOfSpeech=PRP] [Text=still CharacterOffsetBegin=485 CharacterOffsetEnd=490 PartOfSpeech=RB] [Text=is CharacterOffsetBegin=491 CharacterOffsetEnd=493 PartOfSpeech=VBZ] [Text=propaganda CharacterOffsetBegin=494 CharacterOffsetEnd=504 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=504 CharacterOffsetEnd=505 PartOfSpeech=,] [Text=in CharacterOffsetBegin=506 CharacterOffsetEnd=508 PartOfSpeech=IN] [Text=some CharacterOffsetBegin=509 CharacterOffsetEnd=513 PartOfSpeech=DT] [Text=extend CharacterOffsetBegin=514 CharacterOffsetEnd=520 PartOfSpeech=VBP] [Text=- CharacterOffsetBegin=521 CharacterOffsetEnd=522 PartOfSpeech=:] [Text=no CharacterOffsetBegin=523 CharacterOffsetEnd=525 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=526 CharacterOffsetEnd=530 PartOfSpeech=NN] [Text=today CharacterOffsetBegin=531 CharacterOffsetEnd=536 PartOfSpeech=NN] [Text=with CharacterOffsetBegin=537 CharacterOffsetEnd=541 PartOfSpeech=IN] [Text=these CharacterOffsetBegin=542 CharacterOffsetEnd=547 PartOfSpeech=DT] [Text=themes CharacterOffsetBegin=548 CharacterOffsetEnd=554 PartOfSpeech=NNS] [Text=can CharacterOffsetBegin=555 CharacterOffsetEnd=558 PartOfSpeech=MD] [Text=not CharacterOffsetBegin=559 CharacterOffsetEnd=562 PartOfSpeech=RB] [Text=be CharacterOffsetBegin=563 CharacterOffsetEnd=565 PartOfSpeech=VB] [Text=- CharacterOffsetBegin=566 CharacterOffsetEnd=567 PartOfSpeech=:] [Text=but CharacterOffsetBegin=568 CharacterOffsetEnd=571 PartOfSpeech=CC] [Text=it CharacterOffsetBegin=572 CharacterOffsetEnd=574 PartOfSpeech=PRP] [Text=dealt CharacterOffsetBegin=575 CharacterOffsetEnd=580 PartOfSpeech=VBD] [Text=with CharacterOffsetBegin=581 CharacterOffsetEnd=585 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=586 CharacterOffsetEnd=589 PartOfSpeech=DT] [Text=issue CharacterOffsetBegin=590 CharacterOffsetEnd=595 PartOfSpeech=NN] [Text=successfully.a CharacterOffsetBegin=596 CharacterOffsetEnd=610 PartOfSpeech=NN] [Text=film CharacterOffsetBegin=611 CharacterOffsetEnd=615 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=616 CharacterOffsetEnd=620 PartOfSpeech=WDT] [Text=should CharacterOffsetBegin=621 CharacterOffsetEnd=627 PartOfSpeech=MD] [Text=be CharacterOffsetBegin=628 CharacterOffsetEnd=630 PartOfSpeech=VB] [Text=shown CharacterOffsetBegin=631 CharacterOffsetEnd=636 PartOfSpeech=VBN] [Text=in CharacterOffsetBegin=637 CharacterOffsetEnd=639 PartOfSpeech=IN] [Text=all CharacterOffsetBegin=640 CharacterOffsetEnd=643 PartOfSpeech=DT] [Text=terrorism/counter-terrorism CharacterOffsetBegin=644 CharacterOffsetEnd=671 PartOfSpeech=JJ] [Text=courses CharacterOffsetBegin=672 CharacterOffsetEnd=679 PartOfSpeech=NNS] [Text=but CharacterOffsetBegin=680 CharacterOffsetEnd=683 PartOfSpeech=CC] [Text=will CharacterOffsetBegin=684 CharacterOffsetEnd=688 PartOfSpeech=MD] [Text=not CharacterOffsetBegin=689 CharacterOffsetEnd=692 PartOfSpeech=RB] [Text=because CharacterOffsetBegin=693 CharacterOffsetEnd=700 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=701 CharacterOffsetEnd=703 PartOfSpeech=PRP] [Text=shows CharacterOffsetBegin=704 CharacterOffsetEnd=709 PartOfSpeech=VBZ] [Text=faults CharacterOffsetBegin=710 CharacterOffsetEnd=716 PartOfSpeech=NNS] [Text=which CharacterOffsetBegin=717 CharacterOffsetEnd=722 PartOfSpeech=WDT] [Text=is CharacterOffsetBegin=723 CharacterOffsetEnd=725 PartOfSpeech=VBZ] [Text=not CharacterOffsetBegin=726 CharacterOffsetEnd=729 PartOfSpeech=RB] [Text=allowed CharacterOffsetBegin=730 CharacterOffsetEnd=737 PartOfSpeech=VBN] [Text=to CharacterOffsetBegin=738 CharacterOffsetEnd=740 PartOfSpeech=TO] [Text=be CharacterOffsetBegin=741 CharacterOffsetEnd=743 PartOfSpeech=VB] [Text=acknowledged CharacterOffsetBegin=744 CharacterOffsetEnd=756 PartOfSpeech=VBN] [Text=! CharacterOffsetBegin=756 CharacterOffsetEnd=757 PartOfSpeech=.] 
root(ROOT-0, propaganda-4)
nsubj(propaganda-4, it-1)
advmod(propaganda-4, still-2)
cop(propaganda-4, is-3)
punct(propaganda-4, ,-5)
prep(propaganda-4, in-6)
pobj(in-6, some-7)
dep(propaganda-4, extend-8)
punct(extend-8, --9)
det(film-11, no-10)
dobj(extend-8, film-11)
tmod(extend-8, today-12)
prep(extend-8, with-13)
det(themes-15, these-14)
pobj(with-13, themes-15)
dep(extend-8, can-16)
dep(can-16, not-17)
dep(not-17, be-18)
punct(be-18, --19)
cc(not-17, but-20)
nsubj(dealt-22, it-21)
conj(not-17, dealt-22)
prep(dealt-22, with-23)
det(film-27, the-24)
nn(film-27, issue-25)
nn(film-27, successfully.a-26)
pobj(with-23, film-27)
nsubjpass(shown-31, that-28)
aux(shown-31, should-29)
auxpass(shown-31, be-30)
rcmod(film-27, shown-31)
prep(shown-31, in-32)
det(courses-35, all-33)
amod(courses-35, terrorism/counter-terrorism-34)
pobj(in-32, courses-35)
cc(not-17, but-36)
conj(not-17, will-37)
dep(will-37, not-38)
mark(shows-41, because-39)
nsubj(shows-41, it-40)
advcl(not-38, shows-41)
dobj(shows-41, faults-42)
nsubjpass(allowed-46, which-43)
auxpass(allowed-46, is-44)
neg(allowed-46, not-45)
rcmod(faults-42, allowed-46)
aux(acknowledged-49, to-47)
auxpass(acknowledged-49, be-48)
xcomp(allowed-46, acknowledged-49)
punct(propaganda-4, !-50)

Sentence #4 (33 tokens):
A great film in which the BBC took a few risks and unfortunately, London does not need a fictional tale any more, due to the reality of July 7 2005.
[Text=A CharacterOffsetBegin=762 CharacterOffsetEnd=763 PartOfSpeech=DT] [Text=great CharacterOffsetBegin=764 CharacterOffsetEnd=769 PartOfSpeech=JJ] [Text=film CharacterOffsetBegin=770 CharacterOffsetEnd=774 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=775 CharacterOffsetEnd=777 PartOfSpeech=IN] [Text=which CharacterOffsetBegin=778 CharacterOffsetEnd=783 PartOfSpeech=WDT] [Text=the CharacterOffsetBegin=784 CharacterOffsetEnd=787 PartOfSpeech=DT] [Text=BBC CharacterOffsetBegin=788 CharacterOffsetEnd=791 PartOfSpeech=NNP] [Text=took CharacterOffsetBegin=792 CharacterOffsetEnd=796 PartOfSpeech=VBD] [Text=a CharacterOffsetBegin=797 CharacterOffsetEnd=798 PartOfSpeech=DT] [Text=few CharacterOffsetBegin=799 CharacterOffsetEnd=802 PartOfSpeech=JJ] [Text=risks CharacterOffsetBegin=803 CharacterOffsetEnd=808 PartOfSpeech=NNS] [Text=and CharacterOffsetBegin=809 CharacterOffsetEnd=812 PartOfSpeech=CC] [Text=unfortunately CharacterOffsetBegin=813 CharacterOffsetEnd=826 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=826 CharacterOffsetEnd=827 PartOfSpeech=,] [Text=London CharacterOffsetBegin=828 CharacterOffsetEnd=834 PartOfSpeech=NNP] [Text=does CharacterOffsetBegin=835 CharacterOffsetEnd=839 PartOfSpeech=VBZ] [Text=not CharacterOffsetBegin=840 CharacterOffsetEnd=843 PartOfSpeech=RB] [Text=need CharacterOffsetBegin=844 CharacterOffsetEnd=848 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=849 CharacterOffsetEnd=850 PartOfSpeech=DT] [Text=fictional CharacterOffsetBegin=851 CharacterOffsetEnd=860 PartOfSpeech=JJ] [Text=tale CharacterOffsetBegin=861 CharacterOffsetEnd=865 PartOfSpeech=NN] [Text=any CharacterOffsetBegin=866 CharacterOffsetEnd=869 PartOfSpeech=DT] [Text=more CharacterOffsetBegin=870 CharacterOffsetEnd=874 PartOfSpeech=RBR] [Text=, CharacterOffsetBegin=874 CharacterOffsetEnd=875 PartOfSpeech=,] [Text=due CharacterOffsetBegin=876 CharacterOffsetEnd=879 PartOfSpeech=JJ] [Text=to CharacterOffsetBegin=880 CharacterOffsetEnd=882 PartOfSpeech=TO] [Text=the CharacterOffsetBegin=883 CharacterOffsetEnd=886 PartOfSpeech=DT] [Text=reality CharacterOffsetBegin=887 CharacterOffsetEnd=894 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=895 CharacterOffsetEnd=897 PartOfSpeech=IN] [Text=July CharacterOffsetBegin=898 CharacterOffsetEnd=902 PartOfSpeech=NNP] [Text=7 CharacterOffsetBegin=903 CharacterOffsetEnd=904 PartOfSpeech=CD] [Text=2005 CharacterOffsetBegin=905 CharacterOffsetEnd=909 PartOfSpeech=CD] [Text=. CharacterOffsetBegin=909 CharacterOffsetEnd=910 PartOfSpeech=.] 
root(ROOT-0, took-8)
det(film-3, A-1)
amod(film-3, great-2)
nsubj(took-8, film-3)
prep(took-8, in-4)
pobj(in-4, which-5)
det(BBC-7, the-6)
nsubj(took-8, BBC-7)
det(risks-11, a-9)
amod(risks-11, few-10)
dobj(took-8, risks-11)
cc(took-8, and-12)
advmod(need-18, unfortunately-13)
punct(need-18, ,-14)
nsubj(need-18, London-15)
aux(need-18, does-16)
neg(need-18, not-17)
conj(took-8, need-18)
det(tale-21, a-19)
amod(tale-21, fictional-20)
dobj(need-18, tale-21)
dep(more-23, any-22)
advmod(tale-21, more-23)
punct(tale-21, ,-24)
amod(tale-21, due-25)
prep(due-25, to-26)
det(reality-28, the-27)
pobj(to-26, reality-28)
prep(reality-28, of-29)
pobj(of-29, July-30)
num(July-30, 7-31)
num(July-30, 2005-32)
punct(took-8, .-33)

